src/screens/home/HomeScreen.js

```diff
@@ -0,0 +1,74 @@
+import React, { useContext, useEffect } from 'react';
+
+// Components
+import { StatusBar } from 'react-native';
+import {
+  Container,
+  Header,
+  Title,
+  Content,
+  Footer,
+  FooterTab,
+  Button,
+  Left,
+  Right,
+  Body,
+  Icon,
+  Text,
+} from 'native-base';
+
+// Utils
+import { getUser } from '../../utils/storage';
+
+// Context
+import UserContext from '../../contexts/UserContext';
+
+// Vars
+const CONTENT = `This is where any content will go for the home page! I'm not sure what will end up going, but it's going to be good!
+  Use the links at the bottom of the screen to navigate to search or learning pages!`;
+
+const HomeScreen = ({ navigation }) => {
+  const [user, setUser] = useContext(UserContext);
+
+  useEffect(() => {
+    getUser().then(setUser);
+  }, [setUser]);
```

Comment:
Curious if this could be done at the App level, so you don't need to getUser/setUser in each screen...could be I'm missing something, though.